### ISSUE TITLE
Support media query feature `prefers-reduced-transparency`

### ISSUE THREAD

**Issue by czerkies** [czerkies/romanczerkies.31#154]
- https://www.w3.org/TR/mediaqueries-5/#prefers-reduced-transparency
- https://developer.chrome.com/blog/css-prefers-reduced-transparency/

> Non-opaque interfaces can cause headaches or be a visual struggle for various types of vision deficiencies. This is why [Windows, macOS and iOS have system preferences](https://psdtohtml.dev/ideas/accessibility-transparencies-css/#:%7E:text=How%20to%20activate%20transparency%20reduction) that can reduce or remove transparency from the UI.

**Comment by czerkies** [comment]
- Can be combined with `@media (prefers-contrast: more)`
- https://drafts.csswg.org/mediaqueries/#mq-list

**Comment by czerkies** [comment]
Use of subtractive strategy due to the lack of web browsers support.
New issue to followup the browsers integration: #167
